src/utils/client.rs

```diff
@@ -64,6 +64,7 @@ impl ClientBuilder {
             .danger_accept_invalid_hostnames(self.accept_invalid_hostnames)
             .timeout(self.timeout)
             .user_agent(format!("{}/{}", PKG_NAME, PKG_VERSION))
+            .pool_max_idle_per_host(0)
```

Comment:
__Question__: should we keep this change in this PR?

There might be unknown side-effects with this change, so I'd prefer to remove it, but re-add it at a later time if we see this is needed.